types/moment-timezone/moment-timezone-tests.ts

```diff
@@ -1,4 +1,4 @@
-import moment = require('moment-timezone');
+import * as moment from 'moment-timezone';
```

Comment:
This import syntax is not allowed as `moment` exports a function. I suggest to rollback this change.